Test/CreditPaymentTest.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using FluentAssertions;
+using Xunit;
+using System.Linq;
+
+namespace Recurly.Test
+{
+  public class CreditPaymentTest : BaseTest
+  {
+    [RecurlyFact(TestEnvironment.Type.Integration)]
+    public void LookupCreditPayment()
+    {
+      var collection = CreateNewCollection();
+      var transactionUuid = collection.ChargeInvoice.Transactions[0].Uuid;
+      var cancelledCollection = Purchase.Cancel(transactionUuid);
```

Comment:
For my own learnings: why does the purchase need to be canceled? Pre-emptive cleanup?

Reply:
Canceling the purchase authorization triggers a credit payment on the account. Here, we're authorizing a purchase ([it's abstracted under `CreateNewCollection`](https://github.com/recurly/recurly-client-dotnet/blob/v2/Test/BaseTest.cs#L76-L90)), grabbing the transaction uuid, using that uuid to cancel the purchase authorization, then finally fetching the credit payment that was created when we canceled.